### PR TITLE
Update to kubernetes-1.15.4

### PR DESCRIPTION
**Pull request by sslavic**
Towards https://github.com/giantswarm/giantswarm/issues/6302

### CODE REVIEW

File: LICENSE
Comment (rossf7) [inline LICENSE]: It's a pain but please do a seperate commit for the license. No need for approval.

**Review by rossf7 — APPROVED** [review]
LGTM after license. Good call to remove glide. <3

File: LICENSE
Comment (sslavic) [reply LICENSE]: Move them to separate commit and PR https://github.com/giantswarm/k8s-endpoint-updater/pull/24

**Pull request merged**
